app/views/facility_reservations/_admin_reservation_repeat_fields.html.haml

```diff
@@ -0,0 +1,4 @@
+.checkboxControl
+  = f.input :repeats?, as: :boolean, input_html: { data: { disables: ".admin_reservation_repeat_frequency, .admin_reservation_repeat_end_date" } }, wrapper_html: { class: "checkboxControl__checkbox" }
+  = f.input :repeat_frequency, collection: AdminReservation::REPEAT_OPTIONS.map(&:titleize), selected: "Weekly"
+  = f.input :repeat_end_date, input_html: { class: "datepicker" }
```

Comment:
Can't figure out how to get this input to start with the default value of today like the other datepickers on the page do.

Reply:
Not sure quite where the right place to do it without thinking a little more, but you need to set the `repeat_end_date` attribute before you pass it to the partial. The form object initializer might be an ok place: `@reservation_form.repeat_end_date = reservation.reserve_end_at`, or something like that.